src/SourceBuild/tarball/content/patches/roslyn/0002-Don-t-include-desktop-artifacts-that-don-t-exist-in-.patch

```diff
@@ -0,0 +1,56 @@
+From a235579b23a9aaf3f30202a69e92b93f7c569107 Mon Sep 17 00:00:00 2001
+From: Chris Rummel <crummel@microsoft.com>
+Date: Tue, 20 Jul 2021 14:20:14 -0500
+Subject: [PATCH 2/2] Don't include desktop artifacts that don't exist in
```

Comment:
I'd like to see us step up the quality of the patch descriptions.  We should be describing in more detail the issue the patch is addressing.  We should include information why source-build encounters it when the repo SB legs don't.

Reply:
Added comments to the patches.